src/utils/validateMerchantCode.tsx

```diff
@@ -1,4 +1,4 @@
-export const merchantCodeRegex = /^[a-zA-Z0-9 _-]+$/; // Alphanumeric, space, underscore, dash
+export const merchantCodeRegex = /^[a-zA-Z0-9]{1,8}$/;
```

Comment:
Why can't we use policy validationRegex for this?

Reply:
Yea was thinking we stick to this first, once the backend PR merges, we can change it!